### DISCUSSION TITLE
Any Documentation?

### DISCUSSION THREAD

**Discussion by Walker-00** [HFQR/xitca-web#720] [General]
Hey I'm wondering that is the xitca-web has documentation or something like that? I've been seeing xitca-web in [techempower](https://www.techempower.com) for a long time, and I wanna test it now, So it's there any documentation like in [docs](https://docs.rs) or something like that?

**Comment by fakeshadow** [comment]
Sorry there is no online doc right now as this project has never been released on `crates.io`
That said you can clone the project and run `cargo doc` to generate it locally. See [here](https://doc.rust-lang.org/cargo/commands/cargo-doc.html) for some tutorial.
